ecosystem/typescript/sdk_v2/src/crypto/multi_ed25519.ts

```diff
@@ -0,0 +1,159 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+/* eslint-disable no-bitwise */
+import { Deserializer, Serializer } from "../bcs";
+import { Uint8 } from "../types";
+import { Ed25519PublicKey, Ed25519Signature } from "./ed25519";
+
+/**
+ * MultiEd25519 currently supports at most 32 signatures.
+ */
+const MAX_SIGNATURES_SUPPORTED = 32;
```

Comment:
thinking on maybe have it as a static variable in both classes instead of a global const... it doesn't make sny different but easier to understand and use for developers, i.e
```
MultiEd25519PublicKey.MAX_SIGNATURES_SUPPORTED

MultiEd25519Signature.MAX_SIGNATURES_SUPPORTED
```